src/site/topic2.rst

```diff
@@ -817,4 +817,4 @@ For next time
 * Read Chapter 1 of your text
     * 15 pages
 
-* :doc:`If you would like to see how a C++ differs, check out this aside. </topic2-cpp>`
+* :doc:`If you would like to see how a C++ differs, check out the relevant C++ lesson. </cpp-syntax>`
```

Comment:
I'd also specify that it's outside the scope of the course, they don't need to know it.